src/Sdk/DTWebApi/WebApi/ActionsEnvironmentReference.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Runtime.Serialization;
+using GitHub.DistributedTask.ObjectTemplating.Tokens;
+
+namespace GitHub.DistributedTask.WebApi
+{
+    /// <summary>
+    /// Information about an environment parsed from YML with evaluated name, URL will be evaluated on runner
+    /// </summary>
+    [DataContract]
+    public class ActionsEnvironmentReference
+    {
+        public ActionsEnvironmentReference(string name)
+        {
+            Name = name;
+        }
+
+        [DataMember]
+        public string Name { get; set; }
+
+        [DataMember]
+        public TemplateToken Url { get; set; }
```

Comment:
DeployedUrl or InstanceUrl might be a better name (potential future refactor)

Reply:
discussed offline, product decision is to `url` https://github.com/github/c2c-actions/discussions/1694#discussioncomment-105258